lib/src/view/broadcast/broadcast_game_tree_view.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:lichess_mobile/src/model/analysis/analysis_preferences.dart';
+import 'package:lichess_mobile/src/model/broadcast/broadcast_game_controller.dart';
+import 'package:lichess_mobile/src/model/common/id.dart';
+import 'package:lichess_mobile/src/widgets/pgn.dart';
+
+const kOpeningHeaderHeight = 32.0;
+
+class BroadcastGameTreeView extends ConsumerWidget {
+  const BroadcastGameTreeView(
+    this.roundId,
+    this.gameId,
+  );
+
+  final BroadcastRoundId roundId;
+  final BroadcastGameId gameId;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final ctrlProvider = broadcastGameControllerProvider(roundId, gameId);
+
+    final root =
```

Comment:
Here you can also watch the whole state at once, instead of using several `.select`